crates/bevy_ecs/src/system/system_registry.rs

```diff
@@ -43,6 +43,29 @@ pub struct SystemId<I = (), O = ()> {
     pub(crate) marker: std::marker::PhantomData<fn(I) -> O>,
 }
 
+impl<I, O> SystemId<I, O> {
+    /// Transforms a [`SystemId`] into the [`Entity`] that holds the one-shot system's state.
+    ///
+    /// It's trivial to convert [`SystemId`] into an [`Entity`] since a system
+    /// is really an entity with associated handler function.
+    ///
+    /// For example, this is useful if you want to assign a name label to a system.
+    pub fn entity(self) -> Entity {
```

Comment:
I would call this method `id`, like in [Entity commands](https://docs.rs/bevy_ecs/latest/bevy_ecs/system/struct.EntityCommands.html#method.id)

Reply:
`SystemId::id` is a bit confusing. `entity` gives you an expectation of what to receive & is what you would look/search for in the doc page. Where as `id` is... it's already a System**Id**.